crates/viewer/re_viewer/src/ui/rerun_menu.rs

```diff
@@ -166,6 +166,7 @@ impl App {
             is_in_rerun_workspace: _,
             target_triple,
             datetime,
+            is_debug_build: _,
```

Comment:
It would be nice to include this in the about-menu too, so we can ask folks to just screenshot that and have all the info we need